static/pattern-library/sass/main-ltr.scss

```diff
@@ -14,7 +14,7 @@
 // ------------------------------
 // #UTILITIES
 // ------------------------------
-@import 'utilities/variables';
+@import 'utilities/variables-ltr';
```

Comment:
Hey, yeah, there's a separate ``_variables-rtl.scss`` partial in place as well - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static/pattern-library/sass/utilities/_variables-rtl.scss. Its only called from https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static/pattern-library/sass/main-rtl.scss. In short, the PL and all apps using it will now have 3 variable/config-based partials:

* general variables/config (non LTR/RTL dependent)
* LTR settings/config
* RTL settings/config

Let me know if you have any concerns or if that's still fuzzy.